index.html

```diff
@@ -43,7 +43,7 @@
         }, {
             name : "Farshid Tavakolizadeh",
             w3cid : "122520",
-            company : "Fraunhofer-Gesellschaft",
+            company : "formerly Fraunhofer-Gesellschaft, currently Invited Expert",
```

Comment:
I don't think this is the correct way of defining multiple affiliations. There is a w3cid for the company associated with this entry. The rendering hyperlinks the whole value with Fraunhofer's URL.
As mentioned in the call, other specs do this by creating another group for Former Editors.
Examples: https://w3c.github.io/webrtc-pc/ and https://www.w3.org/TR/2016/WD-html51-20160412/

I've also seen one spec having one author both in current and former to separate the affiliations. However I could not find that anymore.

Reply:
let's merge for now, Farshid will make a PR to fix all the way